src/App.js

```diff
@@ -22,7 +22,7 @@ function App() {
           <ProtectedRoute exact path="/dashboard" component={DashBoard} />
           <ProtectedRoute path="/profiles" component={ProfileSelectionPage} />
           <ProtectedRoute path="/welcome" component={WelcomePage} />
-          <ProtectedRoute path="/toiletUsers" component={ToiletUsersPage} />
+          <Route path="/toiletUsers" component={ToiletUsersPage} />
```

Comment:
Any reason why ProtectedRoute is removed?

Reply:
to skip login again